apps/examples/components/README.md

```diff
@@ -32,28 +30,133 @@ For example, one can instantiate a component via:
 where `mycomp` is the path used for the component AJAX callbacks, and `session` is an 
 example of an additional parameter that the component might need. 
 
+### Instantiating a component in a template
+
 Once the component is thus defined, it can be used in a controller, for example via:
 
     @action('mypage')
     @action.uses(session)
     def mypage():
         c = my_component(id=1)
         return dict(my_component=my_component)
-        
+
```

Comment:
Where is c being used here? The line below mentions `my_component(id=1)` which is set to c. Should the return instead be `return dict(my_component=c)` ?

Reply:
Thanks, typo fixed.